metabase/metabase
# Mantine Popover

Ports [Mantine Popover](https://v6.mantine.dev/core/popover/) to `metabase/ui/overlays`

### To Verify

1. `yarn storybook`
2. Open `http://localhost:6006/?path=/story/overlays-popover`
3. Ensure the popover works as expected
